detection_rules/etc/version.lock.json

```diff
@@ -3450,32 +3464,32 @@
       }
     },
     "rule_name": "Attempt to Disable Gatekeeper",
-    "sha256": "b439aeedce5e2ddd54f3d1c6402159147520ca3e3598273f235dbcc01c58a6fb",
+    "sha256": "e626d42c263b5f4f856008238c6128c93f4314fb38f49395bc5f1b7a1e46520e",
     "type": "query",
-    "version": 100
+    "version": 101
   },
   "4de76544-f0e5-486a-8f84-eae0b6063cdc": {
     "min_stack_version": "8.3",
     "previous": {
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "Disable Windows Event and Security Logs Using Built-in Tools",
-        "sha256": "d3c7114cd8a47dc33281f4f4f124e0b3ddb9b16021a2a26d7ac01b8a3ce8dc31",
+        "sha256": "51d1269f8e2276398c0f5d29467e8bdd1f4dbb5235021d0dc5f3b251fb6c39d7",
         "type": "eql",
-        "version": 7
+        "version": 8
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.